Controller/Admin/Transaction/TransactionTab.php

```diff
@@ -103,4 +103,16 @@ protected function _getListDataFromArray(array $aArray, string $sTransactionStat
 
         return $aListData;
     }
+
+    /**
+     * Returns true if the payment is one of the module's.
+     *
+     * @return bool
+     *
+     * @since 1.0.1
+     */
+    public function isCustomPaymentMethod()
```

Comment:
you could move it to Tab base class

Reply:
+1

Reply:
👍 I would also rename it to `shouldDisplayLiveChat`.

Reply:
👍